### ISSUE TITLE
BetterDiscord wont install for me error

### ISSUE THREAD

**Issue by AtoliRose25** [Jiiks/BetterDiscordApp#658]
Having problems with better discord it wont install plus theres 1 error it says 

Killing Discord
Deleting old cached files
Deleting temp path
Downloading Resource: BetterDiscord.zip
Extracting BetterDiscord
Extracting app.asar
Moving BetterDiscord to resources\node_modules\
Error: index.js not found
Finished installing BetterDiscord with 1 errors

how can i fix the error on this?
please help

**Comment by BarterClub** [comment]
Got the same one.

**Comment by ZeLocks542** [comment]
Me too

**Comment by Venaf** [comment]
Same error.

**Comment by AtoliRose25** [comment]
If anyone knows how to resolve this issue would be great cuz i wanna get betterdiscord to work 
:c

**Comment by Soyoomi** [comment]
As stated in previous threads.
Use this one. And make sure to tick the unofficial fork.
Ignore the fact that it says canary, it works for the regular version too. 
https://github.com/rauenzi/BetterDiscordApp/releases

**Comment by BarterClub** [comment]
@Soyoomi  Thank you. For some reason i didn't get the option to do so.

**Comment by AtoliRose25** [comment]
@Soyoomi I tried that its still not working work for me =/

**Comment by AtoliRose25** [comment]
I'm still having the same issue i keep trying betterdiscord again to install but i still get the same 
error.....i'm still trying to solve this problem =/

**Comment by Henbun** [comment]
I get the problem as well.
I can't look at the Zeke fix's code, but it doesn't work for me either.

The main program error seems to lie in discord changing/moving index.js from "app" to "app\\app_bootstrap" and splitting it from the main window.
Current program wont work because it looks in 2 places that the files required are not located.

`string curPath = $"{GetParent().DiscordPath}\\resources\\app\\index.js";
                string curPath2 = $"{GetParent().DiscordPath}\\core\\app\\mainScreen.js";
                 if (!File.Exists(curPath) && !File.Exists(curPath2))
                 {
                     AppendLog($"ERROR: index.js or mainScreen.js not found in {curPath} or {curPath2}");
                     errors++;
                 }`

I haven't looked much further into the changes, but it seems like they also (re)moved the window information that existed in previous versions.

**Comment by TyNotTie-A** [comment]
I'm also having this issue

**Comment by Henbun** [comment]
I made a gif, to show how it happens to me, with the newest available version, though I'm not sure if it's going to help any.

![bd gif 2](https://user-images.githubusercontent.com/35263495/34734811-716895b2-f56d-11e7-8b97-62a0b9ee4ab0.gif)

**Comment by DevilsLynAvenged** [comment]
Try restarting your computer and try again I guess

**Comment by Henbun** [comment]
I restarted my PC before I tried, due to an unrelated crash.
I've had no luck with either the regular, PTB or Canary versions.

**Comment by AtoliRose25** [comment]
BetterDiscord still isnt working for me sadly....  :c

**Comment by DevilsLynAvenged** [comment]
and you also ran as admin?

**Comment by Henbun** [comment]
Regardless of whether I run it as admin or not it fails.
I've tried to many times I stopped counting.

**Comment by AtoliRose25** [comment]
I dont think betterdiscord will ever work for me...its still an issue

**Comment by AtoliRose25** [comment]
how can i still fix so betterdiscord works i want it to work the betterdiscord logo on the corner has been loading for awhile now and the theme hasnt loaded

**Comment by AtoliRose25** [comment]
Oooooh now i got it to work omg xD

**Comment by AtoliRose25** [comment]
but now i cant see the theme in the background...

**Comment by rauenzi** [comment]
@Henbun are you still having that issue?

**Comment by Henbun** [comment]
As if this writing, I am still getting 1 error, missing index.js or mainScreen.js file.

**Comment by Henbun** [comment]
My  issue with installing have been solved by using the modified installer from [issue/667](https://github.com/Jiiks/BetterDiscordApp/issues/667) although I had to change the directory from "AppData\Local\Discord\app-0.0.300" to "AppData\Roaming\Discord\0.0.300".

If I change the directory with the current rauenzi version I get a "missing app.asar" message.
Is it possible that the current version checks to see if app.asar exists in a certain location and prevents installing to "desktop_core" if it does, or am I mistaken here?
